### DISCUSSION TITLE
can you help to make bash

### DISCUSSION THREAD

**Discussion by bellad17** [micropython/micropython#9797] [General]
Hello, i have 12 esp32 modul , every year I have to send a file ' d.txt'  for the time change summer/winter
if i do manuel ( on linux ) `./webrepl_cli.py -p XXXXXXX d.txt 192.168.1.200` , it's ok
i try  to make bash , for automate , but error " One remote file is required "
the bash file ,  webrepl_cli.py , websocket_helper.py , d.txt is in root directory
thank you

```bash
#!/bin/bash

cd /root
webrepl_cli.py -p XXXXX d.txt 192.168.1.200
sleep 10
echo " ok 200"
webrepl_cli.py -p XXXXX d.txt 192.168.1.201
sleep 10
webrepl_cli.py -p XXXXX d.txt 192.168.1.202
sleep 10
webrepl_cli.py -p XXXXX d.txt 192.168.1.203
sleep 10
webrepl_cli.py -p XXXXX d.txt 192.168.1.204
sleep 10
webrepl_cli.py -p XXXXX d.txt 192.168.1.205
sleep 10
webrepl_cli.py -p XXXXX d.txt 192.168.1.206
sleep 10
webrepl_cli.py -p XXXXX d.txt 192.168.1.207
sleep 10
webrepl_cli.py -p XXXXX d.txt 192.168.1.208
sleep 10
webrepl_cli.py -p XXXXX d.txt 192.168.1.209
sleep 10
webrepl_cli.py -p XXXXX d.txt 192.168.1.210
sleep 10
webrepl_cli.py -p XXXXX d.txt 192.168.1.211
sleep 10
```

(Edit by @jimmo to add code formatting)

**Comment by jimmo** [comment]
You are missing the `:` at the end of the remote file name.

e.g.

```bash
webrepl_cli.py -p XXXXX d.txt 192.168.1.200:
```

You can also write

```bash
webrepl_cli.py -p XXXXX d.txt 192.168.1.200:d.txt
```

(This works the same way as `scp` where the colon indicates that it's a remote file)

FYI -- please see https://github.com/orgs/micropython/discussions/9111 for instructions on how to format code snippets.

**Reply by bellad17** [reply]
ok , thank you , it's ok with  `./webrepl_cli.py -p XXXXX d.txt 192.168.1.200:`
